dask_kubernetes/operator/tests/test_operator.py

```diff
@@ -77,6 +77,12 @@ async def test_simplecluster(k8s_cluster, kopf_runner, gen_cluster):
                 async with Client(
                     f"tcp://localhost:{port}", asynchronous=True
                 ) as client:
+                    k8s_cluster.port_forward(
+                        "scale",
+                        "--replicas=5",
+                        "daskworkergroup",
+                        "default-worker-group",
+                    )
```

Comment:
I think we should make a separate test for this instead of extending the existing test.